### ISSUE TITLE
Does the img_fake_att of decoder represent x_^ in the paper?

### ISSUE THREAD

**Issue by chengxiqiao** [ZhaoJ9014/High-Performance-Face-Recognition#11]
If it does, the code "return x, img * att + x * (1. - att), att"  is contrary to the Eq.(13) in paper

**Comment by chengxiqiao** [comment]
Is the code complete? I'm curious about how to keep the identity information @ZhaoJ9014

**Comment by Hzzone** [comment]
img_fake_att is the attention mask which is first proposed in [GANimation: Anatomically-aware Facial Animation from a Single Image, eccv 2018]. I am confused about why the code in https://github.com/ZhaoJ9014/High-Performance-Face-Recognition/blob/a0b6b6b269092051afb7a09a0100409cbccfa493/src/Look%20Across%20Elapse-%20Disentangled%20Representation%20Learning%20and%20Photorealistic%20Cross-Age%20Face%20Synthesis%20for%20Age-Invariant%20Face%20Recognition.TensorFlow/network.py#L150 has fed it into D_img which makes no sense....

**Comment by Hzzone** [comment]
Besides, I have not found $L_{ip}$ in this code.....😥😥😥😥
